src/spasht/ui/template/agent_spasht_search.html.twig

```diff
@@ -7,9 +7,38 @@
 #}
 
 <form id="searchForm" method="post" action="">
-  <input type="hidden" name="formName" value="SearchForm" />
+  {# <input type="hidden" name="formName" value="SearchForm" /> #}
   Search: <input type="text" name="patternName" id="searchBox" value={{ uploadName }} size="30"/>

```

Comment:
How about, changing the **Search:** to **Name:** so that it blends with the advance form better and renaming the **submit** button to **Search**?

Reply:
Changed in new commits